From e36ba573e0dda566bf3c27fb055ec02845408b7f Mon Sep 17 00:00:00 2001
From: sxyzy1016 <sxyzy1016@outlook.com>
Date: Thu, 2 Apr 2026 10:38:16 +0800
Subject: [PATCH 3/3] Add SATMOS to remote configs

---
 src/config/remote-configs.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/config/remote-configs.js b/src/config/remote-configs.js
index 26ad2d5..153b5b8 100644
--- a/src/config/remote-configs.js
+++ b/src/config/remote-configs.js
@@ -16,6 +16,7 @@ export const REMOTE_CONFIGS = [
   {
     label: "customized",
     options: [
+      { label: "SATMOS", value: "https://cdn.jsdelivr.net/gh/sxyzy1016/sub-ini@main/tsutsu-full-tag-router.ini" },
       { label: "Maying", value: "https://cdn.jsdelivr.net/gh/SleepyHeeead/subconverter-config@master/remote-config/customized/maying.ini" },
       { label: "Ytoo", value: "https://cdn.jsdelivr.net/gh/SleepyHeeead/subconverter-config@master/remote-config/customized/ytoo.ini" },
       { label: "FlowerCloud", value: "https://cdn.jsdelivr.net/gh/SleepyHeeead/subconverter-config@master/remote-config/customized/flowercloud.ini" },
-- 
2.43.0

